### DISCUSSION TITLE
sync-ab-with-recent-sessions

### DISCUSSION THREAD

**Discussion by mahescho** [rustdesk/rustdesk-server-pro#313] [Q&A]
In advanced settings I found:

sync-ab-with-recent-sessions
Controls whether to sync the address book with recent sessions.
Location:
Desktop Home → Peer panel → Address book → Tags → Dropdown menu → Sync with recent sessions

I've no clue what this does :-)

**Comment by 21pages** [comment]
Sync username, hostname, platform to address book from recent peers

**Reply by mahescho** [reply]
Thanks, I will give it a try.
